### PR TITLE
added exception handling as hotfix of #6696

### PR DESCRIPTION
**Pull request by gardian12**

### CODE REVIEW

**Review by larshp — APPROVED** [review]
looking good, `cx_no_check` already used in `zcl_abapgit_data_utils`, so I guess its okay for 702

**Comment by larshp** [comment]
looking good, just leaving it open to let others have a chance to review

will merge tomorrow if nobody has any comments

**Comment by gardian12** [comment]
Hello @larshp, short question regarding the merge. Who will do it and probably when?
Kind Regards,
gardian12

**Comment by larshp** [comment]
> Hello @larshp, short question regarding the merge. Who will do it and probably when?

I'll do it, and probably tomorrow

File: src/data/zcl_abapgit_data_utils.clas.abap [hunk 1]
```diff
@@ -168,6 +168,11 @@ CLASS zcl_abapgit_data_utils IMPLEMENTATION.
         lv_contflag = <ls_any>.
       CATCH cx_sy_dyn_call_illegal_class.
```
Comment (mbtools) [inline src/data/zcl_abapgit_data_utils.clas.abap:169]: Isn't one catch sufficient?
```abap
CATCH cx_sy_dyn_call_illegal_class cx_no_check.
  SELECT SINGLE contflag FROM ('DD02L') INTO lv_contflag WHERE tabname = iv_name.
```

Comment (larshp) [reply src/data/zcl_abapgit_data_utils.clas.abap:169]: ahh, yea @gardian12 can you move the catch so there is only one?

Comment (gardian12) [reply src/data/zcl_abapgit_data_utils.clas.abap:169]: 👍
Oh, yes. Wasn't aware of listen up the exceptions at one catch.
And yes, this is the better approach not to duplicate the code inside catch block.

I'll do the change.

**Comment by gardian12** [comment]
Simplification done.
Thanks @mbtools

**Comment by larshp** [comment]
welcome to abapGit as a contributor 🎉

**Pull request merged**